### PR TITLE
cloud wallet docs

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: integration-tests/cloud-wallet.test.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    });
+
+    await pullDocuments();
+
+    const sdkDocument = await dataStore.documents.getDocumentById(newDocId);
+
+    expect(sdkDocument).toEqual(newDoc);
+  });
+
+  // it('should sync a document added to the wallet to the EDV', async () => {
```
Comment (cykoder) [inline integration-tests/cloud-wallet.test.ts:78]: any reason all below here is commented?

Comment (maycon-mello) [reply integration-tests/cloud-wallet.test.ts]: it was a mistake, I've already uncommented that in a separate PR

**Comment by maycon-mello** [comment]
I've got tests working after some code changes in the cloud wallet implementation. We had an event handler being called twice and trying to insert the same document multiple times

File: docs/cloud-wallet.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+```ts
+import {initializeCloudWallet} from '@docknetwork/wallet-sdk-core/src/cloud-wallet';
+import {createWallet} from '@docknetwork/wallet-sdk-core/src/wallet';
+import {edvService} from '@docknetwork/wallet-sdk-wasm/src/services/edv';
+```
+
+### Step 2: Initialize the Data Store
+
+For **Mobile** and **Node.js**:
```
Comment (mike-parkhill) [inline docs/cloud-wallet.md:31]: should these be a heading or bolded?

File: docs/cloud-wallet.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+# Cloud Wallet Documentation
+
+The Cloud Wallet feature allows secure storage and synchronization of wallet documents via an Encrypted Data Vault (EDV).
+
+**File Path:** `@docknetwork/wallet-sdk-core/src/cloud-wallet`
+
+## Feature Overview
+
+The Cloud Wallet integrates with an EDV to securely store, sync, and manage documents. Once initialized, it automatically synchronizes documents between the EDV and the wallet, allowing you to add, update, remove, without dealing with the synchronization logic.
```
Comment (mike-parkhill) [inline docs/cloud-wallet.md:13]: should we link out to what an EDV is? Or do you think it's safe to assume people know what you're talking about?

**Review by mike-parkhill — APPROVED** [review]

**Pull request merged**